crates/ruma-client-api/src/r0/profile/get_avatar_url.rs

```diff
@@ -45,7 +54,7 @@ impl<'a> Request<'a> {
 
 impl Response {
     /// Creates a new `Response` with the given avatar URL.
-    pub fn new(avatar_url: Option<MxcUri>) -> Self {
-        Self { avatar_url }
+    pub fn new(avatar_url: Option<MxcUri>, blurhash: Option<String>) -> Self {
+        Self { avatar_url, blurhash }
```

Comment:
This constructor should not have a second argument, it should just set `blurhash` to `None` (under the pre-spec flag, otherwise it doesn't exist).